systems/subscriber/registry/pkg/server/subscriber.go

```diff
@@ -26,37 +23,38 @@ type SubcriberServer struct {
 	subscriberRepo       db.SubscriberRepo
 	msgbus               mb.MsgBusServiceClient
 	subscriberRoutingKey msgbus.RoutingKeyBuilder
-	pb.UnimplementedSubscriberRegistryServiceServer
-	simManagerService clientPkg.SimManagerClientProvider
-	network        client.Network
-
+	pb.UnimplementedRegistryServiceServer
+	simManagerService client.SimManagerClientProvider
+	network           client.Network
 }
 
-func NewSubscriberServer(subscriberRepo db.SubscriberRepo, msgBus mb.MsgBusServiceClient, simManagerService clientPkg.SimManagerClientProvider, network client.Network) *SubcriberServer {
+func NewSubscriberServer(subscriberRepo db.SubscriberRepo, msgBus mb.MsgBusServiceClient, simManagerService client.SimManagerClientProvider, network client.Network) *SubcriberServer {
 	return &SubcriberServer{subscriberRepo: subscriberRepo,
 		msgbus:               msgBus,
 		simManagerService:    simManagerService,
-		network:  network,
+		network:              network,
 		subscriberRoutingKey: msgbus.NewRoutingKeyBuilder().SetCloudSource().SetContainer(pkg.ServiceName)}
 }
 
 func (s *SubcriberServer) Add(ctx context.Context, req *pb.AddSubscriberRequest) (*pb.AddSubscriberResponse, error) {
 	logrus.Infof("Adding subscriber: %v", req)
-	networkID := uuid.FromStringOrNil(req.GetNetworkID())
-	orgID := uuid.FromStringOrNil(req.GetOrgID())
-	if networkID == uuid.Nil {
-		return nil, status.Errorf(codes.InvalidArgument, "NetworkID must not be empty")
+	networkID, nUuidErr := uuid.FromString(req.GetNetworkID())
+	if nUuidErr != nil {
+		return nil, status.Errorf(codes.InvalidArgument,
+			"invalid format of network uuid. Error %s", nUuidErr.Error())
 	}
-	if orgID == uuid.Nil {
-		return nil, status.Errorf(codes.InvalidArgument, "OrgID must not be empty")
+	orgID, orgUuidErr := uuid.FromString(req.GetOrgID())
+
+	if nUuidErr != nil {
+		return nil, status.Errorf(codes.InvalidArgument,
+			"invalid format of org uuid. Error %s", orgUuidErr.Error())
```

Comment:
`nUUIDErr` and `orgUUIDErr` are only used if one if statement and you don't have to personalize them because they have no impact further down the road. using and reusing `err` will do fine here.